touchforms/backend/decorators.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import with_statement
+from functools import wraps
+
+
+def modify_xform_session(fn):
```

Comment:
Perhaps this could be called `require_xforms_session` since it injects the session into the function.